maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,200 @@
+                generation.result(OpenAIUtils.parse_completion(response))
+            if is_local_trace and trace is not None:
+                trace.set_output(response.choices[0].message.content or "")
+                trace.end()
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][MaximPortkeyChatCompletions] Error in logging generation: {str(e)}"
+            )
+
+        return response
+
+    def __getattr__(self, name):
```

Ruff (0.11.9)

Missing return type annotation for special method `__getattr__`

(ANN204)